lib/src/main/java/org/team100/lib/localization/Blip24ArrayListener.java

```diff
@@ -42,7 +42,7 @@ void consumeValues(NetworkTableEvent e) {
                 System.out.println(fields[1] + " " + blip);
             }
         } else {
-            System.out.println("weird vision update key: " + name);
+            // System.out.println("weird vision update key: " + name);
```

Comment:
you should probably look at why this is happening, instead of just commenting it out.